services/website/website.service.js

```diff
@@ -95,9 +95,17 @@ module.exports = class Website extends LegacyService {
   static get examples() {
     return [
       {
-        previewUrl: 'website-up-down-green-red/https/shields.io',
-        queryParams: { label: 'my-website' },
-        keywords: ['website'],
+        title: 'Website',
+        pattern: 'website-up-down-green-red/:protocol/:hostAndPath',
```

Comment:
Should we use `:protocol(https|http)` here too for consistency? I noticed the above Twitter service had it that way

Reply:
Updated!